src/repositories/ColorPaletteRepository.ts

```diff
@@ -55,7 +55,7 @@ export const deleteColorPalette = async (colorPaletteId: number): Promise<void>
     }
 };
 
-export function mapColorPaletteApiToFrontifyColorPalette(colorPaletteApi: ColorPaletteApi): FrontifyColorPalette {
+export function mapColorPaletteApiToFrontifyColorPalette(colorPaletteApi: ColorPaletteApi): ColorPalette {
```

Comment:
We should update the method name as well:
```suggestion
export function mapColorPaletteApiToColorPalette(colorPaletteApi: ColorPaletteApi): ColorPalette {
```